tests/integration/networks_test.py

```diff
@@ -99,3 +133,27 @@ def test_feedforward_network(feedforward_network: Callable) -> None:
     """Feedforward network smoke test"""
 
     assert feedforward_network()
+
+
+def test_feedforward_network_with_obs(
```

Comment:
This would probably be an excessive test, but would be a good check to see if the network can handle a 2D observation being passed through now. Just pass something 2D through and assert that no error is thrown?

Feel free to ignore this one as it's a bit excessive